src/terminal/parser/stateMachine.cpp

```diff
@@ -1961,7 +1976,7 @@ void StateMachine::ProcessString(const std::wstring_view string)
             // If the engine doesn't require flushing at the end of the string, we
             // want to cache the partial sequence in case we have to flush the whole
             // thing to the terminal later.
-            _cachedSequence = _cachedSequence.value_or(std::wstring{}) + std::wstring{ _run };
+            _cachedSequence = _cachedSequence.value_or(std::wstring{}) + std::wstring{ run };

```

Comment:
This is a bit confusing... I thought you can append a string view directly to a string?

Reply:
Nein. Der Fehler ist:

```
error C2676: binary '+': 'std::basic_string<wchar_t,std::char_traits<wchar_t>,std::allocator<wchar_t>>' does not define this operator or a conversion to a type acceptable to the predefined operator
```

Reply:
An alternative:
```cpp
if (!_cachedSequence) {
    _cachedSequence.emplace({});
}

auto& cachedSequence = *_cachedSequence;
cachedSequence.append(_run);
```

Reply:
Yeah, I would rather do it leonard's weird way instead of creating a new full copy of the string just to use in an append operation :smile:

Reply:
Weird way it is!